src/main/kotlin/com/todo/deoji/infrastructure/global/jwt/adapter/ParseTokenAdapter.kt

```
@@ -0,0 +1,69 @@
+        const val ROLE = "role"
+        const val PREFIX = "Bearer "
+    }
+
+    fun parseToken(token: String): String? =
+        if (token.startsWith(JwtPrefix.PREFIX)) token.substring(JwtPrefix.PREFIX.length) else null
+
+    fun getAuthentication(token: String): Authentication {
+        val claims = getClaims(token, jwtProperty.secretKey)
+
+        if (claims.header[Header.JWT_TYPE] != JwtPrefix.ACCESS)
+            throw RuntimeException()
```

detekt (1.23.7)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)